packages/oid4vci-demo-frontend/src/translationDiffrences.js

```diff
@@ -0,0 +1,12 @@
+const fs = require('fs');
+
+const enTranslations = JSON.parse(fs.readFileSync('public/locales/development/en/translation.json', 'utf8'));
+const nlTranslations = JSON.parse(fs.readFileSync('public/locales/development/nl/translation.json', 'utf8'));
+const enKeys = Object.keys(enTranslations);
+const nlKeys = Object.keys(nlTranslations);
+
+const missingInDutch = enKeys.filter(key => !nlKeys.includes(key));
+const missingInEnglish = nlKeys.filter(key => !enKeys.includes(key));
+
+console.log("Missing in Dutch:", missingInDutch);
```

Comment:
Why is this in js and not TS?, also this seems more like something for a ticket

Reply:
yeah this can be a new ticket, but it was so hard to check the translations every time that I decided to create this here. removing this and creating a [ticket](https://sphereon.atlassian.net/browse/VDX-302) for it